package.json

```diff
@@ -50,10 +50,14 @@
   "devDependencies": {
     "animation-frame": "^0.2.4",
     "autoprefixer": "^5.0.0",
-    "babel": "^6.5.2",
+    "babel-cli": "^6.24.1",
     "babel-core": "^6.21.0",
     "babel-loader": "^6.2.10",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-plugin-transform-class-properties": "6.24.1",
+    "babel-plugin-transform-react-jsx": "6.24.1",
+    "babel-preset-es2015": "6.24.1",
+    "babel-preset-stage-0": "6.24.1",
```

Comment:
I did, it wouldn't build without this for me, I suppose you had some packages globally installed?

Now, the tests are still failing for me, but at least it builds.

I can make a PR with only the change in cloneWithRefs if need be